otomi-quickstart-rabbitmq/values.yaml

```diff
@@ -0,0 +1,103 @@
+# Default values for otomi-quickstart-rabbitmq.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullSecrets: []
+nameOverride: ""
```

Comment:
Is the nameOverride required? We only use the fullnameOverride to automatically add the name of the workload

Reply:
It does get used in the _helper.tpl, all the other charts also have nameOverride.